modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheAbstractMetricsSelfTest.java

```diff
@@ -1460,6 +1460,48 @@ public void testRemoveTime() {
         assertEquals(1, Arrays.stream(m.value()).filter(v -> v == 1).count());
     }
 
+    /** */
+    @Test
+    public void testCommitTime() {
```

Comment:
Can we move these tests in `GridCacheTransactionalAbstractMetricsSelfTest`?

Reply:
Done.